# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/StructuredCloneTags.h
# Commit: 0f9f2acf6e13
# Full Hash: 0f9f2acf6e136933da9a105df64ca4cc20bcb15c
# Author: Kagami Sascha Rosylight <krosylight@mozilla.com>
# Date: 2022-04-20 03:39:48
# Regressor Bug: 1561357
# File Overlap Count: 1
# Description:
#   Bug 1561357 - Implement [Serializable] for DOMException r=smaug,sfink
#   
#   Skipping stack serialization here as
#   
#   * it's optional per the spec
# ==============================================================================

diff -r 6e296387dc26 -r 0f9f2acf6e13 dom/base/StructuredCloneTags.h
--- a/dom/base/StructuredCloneTags.h	Tue Apr 19 19:13:08 2022 +0000
+++ b/dom/base/StructuredCloneTags.h	Tue Apr 19 19:31:27 2022 +0000
@@ -96,7 +96,7 @@
 
   // IMPORTANT: Don't change the order of these enum values. You could break
   // IDB.
-  EMPTY_SLOT_8,
+  SCTAG_DOM_DOMEXCEPTION,
 
   // IMPORTANT: Don't change the order of these enum values. You could break
   // IDB.